.gitignore

```diff
@@ -7,4 +7,18 @@ _datafiles/**/plugin-data/*
 vendor/
 server.crt
 server.key
-__debug*
\ No newline at end of file
+__debug*
+<<<<<<< HEAD
+logs/log.txt
+.DS_Store
+=======
+>>>>>>> 0de7238d1a7d497014d4cab3dc5b0554a0c16019
+mud.sh
+WillowdaleMUD
+WillowdaleMUD.pid
+_datafiles/willowdale-config.yaml
+<<<<<<< HEAD
+=======
+logs/*
+*.DS_Store
+>>>>>>> 0de7238d1a7d497014d4cab3dc5b0554a0c16019
```

Comment:
This has merge data in it... Also, Since this is your repo specific this should all be inside of `.git/info/exclude` so that it doesn't get committed to the main repo